multinode-demo/validator.sh

```diff
@@ -11,7 +11,9 @@ set -x
 
 rsync -v -e ssh "$LEADER"/{mint-demo.json,leader.json,genesis.log} . || exit $?
 
-sudo sysctl -w net.core.rmem_max=26214400
+if [[ "$(uname)" = "Linux" ]]; then
```

Comment:
since this doesn't fix anything for WSL, it's just more code, recommend removing before merge

also:
no need to surround $(uname) in quotes, you're inside the bash builtin [[, where you're safe from spaces

Reply:
It removes an unneeded sudo on macOS so it is still a net improvement.  I'll remove the quotes, thanks for that tip

Reply:
consider

```[[ test ]] && sudo```

?

Reply:
✔️